lib/perl/Genome/InstrumentData/Command/MergeAlignments.pm

```diff
@@ -108,6 +113,7 @@ sub params_for_merged_alignment {
         duplication_handler_params => $self->duplication_handler_params,
         duplication_handler_version => $self->duplication_handler_version,
         samtools_version => $self->samtools_version || undef,
+        bedtools_version => $self->bedtools_version || undef,
```

Comment:
Can you remind me why we often have the "|| undef" added to these param hashes?  Wouldn't it jus be undef anyway?